src/server/src/velvet_dawn/dao/__init__.py

```diff
@@ -2,9 +2,29 @@
 
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
+from pathlib import Path
+from os import remove
 
 import velvet_dawn
 
+db_path=Path(__file__).parent.parent.parent / "instance" / "game.db"
+if db_path.exists():
+    i = 0
```

Comment:
Use of `i` is unnecessary 

while true:
  if ...
     break
   elif ...
     break
    else
       ....